cmd/fork-cleaner/main.go

```diff
@@ -35,6 +35,10 @@ func main() {
 			Usage:  "Base GitHub URL",
 			Value:  "https://api.github.com/",
 		},
+		cli.StringFlag{
+			Name:  "user, u",
```

Comment:
maybe `handle` instead?

Reply:
If you prefer

Reply:
Short option would be `h` though...

Reply:
h would conflict with help...

Reply:
So what do you suggest? I stay with `user`?